yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/AttributeConverter.java

```diff
@@ -87,17 +87,19 @@ private Object convertSpecialType(AttrProto.Attribute attribute, Object value) t
             case AttributeConstants.HumanProfileAttributes.DOCUMENT_DETAILS:
                 return documentDetailsAttributeParser.parseFrom((String) value);
             case AttributeConstants.HumanProfileAttributes.DOCUMENT_IMAGES:
-                return filterForImages((List<Object>) value);
+                return filterForImages((List<Object>) value, Image.class, AttributeConstants.HumanProfileAttributes.DOCUMENT_IMAGES);
             default:
                 return value;
         }
     }
 
-    private List<Image> filterForImages(List<Object> values) {
-        List<Image> list = new ArrayList<>();
+    private <T> List<T> filterForImages(List<Object> values, Class<T> clazz, String attributeName) {
```

Comment:
Since this is now generic, should we call it something like `filterForType`?

Reply:
Well spotted, thank you.  Yes we should.